__init__.py

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env python3
```

Comment:
thanks, but this __init__.py is reduntant for me. Why not to add the other methods or functions? It's hard to think what people could need, so they can import them by themselves.